frontend/src/reducers/property.js

```diff
@@ -2,34 +2,39 @@ const initialState = {
   error: "",
   fields: {
     id: "",
-    name: "",
+    landlordId: "",
+    navigatorId: "",
+    propertyName: "",
     address: "",
-    unitNum: "",
-    floorNum: "",
+    unitAptNum: "",
+    floor: 0,
     zipCode: "",
-    type: "",
-    numBeds: 0,
-    numBaths: 0,
-    numBathsShared: 0,
-    basement: false,
-    schoolDist: "",
-    busStop: false,
-    wheelchairAcc: false,
+    housingType: "",
+    bedrooms: 0,
+    bathrooms: 0,
+    sharedBathrooms: 0,
+    hasBasement: false,
+    schoolDistrict: "",
+    busLine: false,
+    wheelchairAccessibility: false,
     elevator: false,
-    dateAvail: "",
-    monthsAvail: 0,
-    rentPerMonth: 0,
-    appFee: 0,
+    dateFirstAvailable: new Date().toString(),
+    yearAvailable: new Date().getFullYear(),
+    potentialMonthAvailable: 0,
+    listingDate: new Date().toString(),
+    whereListed: "",
+    contactMethod: [],
+    lastContactDate: new Date().toString(),
+    lastContactedBy: "",
+    monthlyRent: 0,
+    applicationFee: 0,
     deposit: 0,
-    lastMonthRent: false,
-    CSRAP: false,
-    HVC: false,
-    MSV: false,
-    voucher4: false,
-    voucher5: false,
-    creditCompany: "",
-    backgroundCompany: "",
-    cleanRecord: false,
+    lastMonthRentRequired: false,
+    voucherTypeAccepted: [],
+    voucherTypeNotAccepted: ["MSV", "HCV", "CSRAP", "Voucher 4", "Voucher 5"],
+    creditScreeningCompany: "",
+    backgroundScreeningCompany: "",
+    allowCriminalRecords: false,
     notes: "",
```

Comment:
are all these fields represented? They need to be in the same format (e.g. `year_listed`)
```json
{
  "address": "string",
  "allow_criminal_records": true,
  "application_fee": 0,
  "background_screening_company": "string",
  "bathrooms": 0,
  "bedrooms": 0,
  "bus_line": true,
  "contact_method": [
    "string"
  ],
  "credit_screening_company": "string",
  "date_first_available": "string",
  "deposit": 0,
  "elevator": true,
  "floor": 0,
  "has_basement": true,
  "housing_type": "string",
  "is_available": true,
  "landlord_id": "string",
  "last_contact_date": "string",
  "last_contacted_by": "string",
  "last_month_rent_required": true,
  "listing_date": "string",
  "monthly_rent": 0,
  "navigator_id": "string",
  "potential_month_available": 0,
  "property_name": "string",
  "school_district": "string",
  "shared_bathrooms": 0,
  "unit_apt_no": "string",
  "voucher_type_accepted": [
    "string"
  ],
  "voucher_type_not_accepted": [
    "string"
  ],
  "wheelchair_accessibility": true,
  "where_listed": [
    "string"
  ],
  "year_available": 0,
  "zip_code": "string"
}
```